docs/source/deploy/sandbox.rst

```diff
@@ -0,0 +1,59 @@
+.. Copyright (c) 2020 The DAML Authors. All rights reserved.
```

Comment:
Why is this in `sandbox.rst` and referenced via `deploy-sandbox` if it’s not actually about sandbox?

Reply:
fixed. that was because previously the file was specific to sandbox.